File: GarageWizardAPI/GarageCoreMVC/Models/RoleAccess.cs
using System.Security.Permissions;

namespace GarageCoreMVC.Models
{
    public class RoleAccess
    {
        public string UserID { get; set; } = string.Empty;
        public string Role { get; set; } = string.Empty;
        public bool IsAuthorized { get; set; } = false;
    }
}

File: GarageWizardClient/src/app/models/role-access.ts
export class RoleAccess
{
    UserID!: string;
    Role!: string;
    IsAuthorized!: boolean;

    public constructor(uid: string, role: string, isAuth: boolean = false){
        this.UserID = uid;
        this.Role = role;
        this.IsAuthorized = isAuth;
    }
}